# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: 22649272ffae
# Full Hash: 22649272ffae2ec3077c452cf5acce0326758623
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - move DOM module loader into own file; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137531
# ==============================================================================

diff -r e3fe772947bf -r 22649272ffae dom/script/moz.build
--- a/dom/script/moz.build	Fri Feb 18 12:45:51 2022 +0000
+++ b/dom/script/moz.build	Fri Feb 18 12:45:51 2022 +0000
@@ -20,6 +20,7 @@
 EXPORTS.mozilla.dom += [
     "AutoEntryScript.h",
     "LoadedScript.h",
+    "ModuleLoader.h",
     "ModuleLoaderBase.h",
     "ModuleMapKey.h",
     "ScriptDecoding.h",
@@ -33,6 +34,7 @@
 UNIFIED_SOURCES += [
     "AutoEntryScript.cpp",
     "LoadedScript.cpp",
+    "ModuleLoader.cpp",
     "ModuleLoaderBase.cpp",
     "ModuleLoadRequest.cpp",
     "ModuleMapKey.cpp",
